### PR TITLE
Add launcher functionality to the new dashboard

### PR DESCRIPTION
**Pull request by m00nwtchr**
Mostly done, but not sure what needs to be adjusted for windows packaging, and i need to add the restart button.

### CODE REVIEW

File: alvr/dashboard/src/launcher/mod.rs [hunk 1]
```diff
@@ -0,0 +1,29 @@
+mod commands;
+
+use std::{sync::mpsc, thread, time::Duration};
+
+use crate::WorkerMsg;
+
+pub fn launcher_thread(rx1: mpsc::Receiver<WorkerMsg>) {
```
Comment (Kirottu) [inline alvr/dashboard/src/launcher/mod.rs:7]: Why is the launcher code being run in a separate thread?

Comment (m00nwtchr) [reply alvr/dashboard/src/launcher/mod.rs:7]: Oh, it seemed wrong to do it in the gui thread to me, but i guess nothing is happening in the ui at that time anyway.

**Comment by zarik5** [comment]
Please do not delete the old launcher just yet. The new dashboard is not on par with the old one in terms of functionality and aesthetics.

**Comment by m00nwtchr** [comment]
> Please do not delete the old launcher just yet. The new dashboard is not on par with the old one in terms of functionality and aesthetics.

The old dashboard is still there even without the launcher, though

**Comment by Vixea** [comment]
@zarik5 ready?

**Pull request merged**